### PR TITLE
Edit account

### PR DESCRIPTION
**Pull request by Peter0892**
Added more page as well as basic edit account form

### CODE REVIEW

**Comment by ghiobi** [comment]
Does this feature also include HTTP calls to the sever? Or did you want to just pull for front-end of edit account?

If so please, put all your template in the html file and add spaces in the imports.

File: src/pages/edit-account/edit-account.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!--
+  Generated template for the EditAccount page.
```
Comment (Scieon) [inline src/pages/edit-account/edit-account.html]: Should remove auto-generated comments since they will affect our quality metrics

**Comment by ghiobi** [comment]
@Peter0892 Are you still working on this, please give us updates.

**Review by ghiobi — APPROVED** [review]
Great thanks peter.

**Pull request merged**